src/pages/Blogs/singleBlog.tsx

```diff
@@ -44,6 +44,15 @@ const SingleBlogView = () => {
       dispatch(getBlogById(id));
   }, [dispatch]);
 
+  const handleDeleteBlog = async (id) => {
+    try {
+      await dispatch(deleteBlogAction(id));
+      window.location.href = "/blogs#/applicant/blogs";
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
```

Comment:
- Remove all logs
- This implementation seems to be similar with previous 
- You should find a way to redirect user when request success

Reply:
i've fixed all the issues, please review again